index.html

```diff
@@ -1238,6 +1238,318 @@ <h2>Evidence</h2>
+
+<p>
+If the subject is not forbidden by the issuer to pass the whole or part of a verifiable credential to a holder, for example, a patient may pass a prescription verifiable credential to a relative for the relative to present it to a pharmacist for dispensing, then the subject MUST issue a verifiable credential to the holder containing the claim properties that are being passed on, as described below. </p>
+
+   <section>
+
+   <h3> 'Subject Only' Terms of Use </h3>
+
+  <p>
+The Subject Only Terms of Use states that a verifiable credential MUST only be presented to a verifier by the subject. If a verifier is presented with a verifiable credential containing the Subject Only Terms of Use, by anyone other than the subject, it MUST refuse to accept it.</p>
+
+        <pre class="example nohighlight" title="Subject Only termsOfUse property by an Issuer">
+{
+  "id": "http://dmv.example.gov/credentials/3732",
+  "type": ["VerifiableCredential", "ProofOfAgeCredential"],
+  "issuer": "https://dmv.example.gov/issuers/14",
+  "issued": "2010-01-01T19:73:24Z",
+  "claim": {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    "ageOver": 21
+  },
+  <span class="highlight">"termsOfUse": [{
+    "type": "SubjectOnly",
+    }]
+  }</span>,
+  "proof": { ... }
+}
+        </pre>
+
+       </section>
+       
+  <section>
+   <h3> Passing on a Verifiable Credential </h3>
+
+   <p>
+
+When the subject passes a verifiable credential to a
+holder, the subject SHOULD issue a new verifiable credential to the holder in which: 
+the issuer is the subject, 
+the subject is the holder to whom the verifiable credential is being passed, 
+and the claim contains the properties that are being passed on. In addition, the holder creates a verifiable presentation that contains these two
+verifiable credentials.
+</p>
+
+
+
+        <pre class="example nohighlight" title="An example of a holder presenting verifiable
+credential properties that have been passed to it by the subject">
+{
+	"id": "did:example:76e12ec21ebhyu1f712ebc6f1z2",
+	"type": ["VerifiablePresentation"],
+	"credential": [{
+			"id": "http://example.gov/credentials/3732",
+			"type": ["Credential", "PrescriptionCredential"],
+			"issuer": "https://dmv.example.gov",
+			"issued": "2010-01-01",
+			"claim": {
+				"id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+				"prescription": {....}
+			},
+			"revocation": {
+				"id": "http://example.gov/revocations/738",
+				"type": "SimpleRevocationList2017"
+			},
+			"proof": {....}
+		},
+		{
+			"id": "https://example.com/VC/123456789",
+			"type": ["Credential", "PrescriptionCredential"],
+			"issuer": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+			"issued": "2010-01-03",
+			"claim": {
+				"id": "did:example:76e12ec21ebhyu1f712ebc6f1z2",
+				"prescription": {....}
```

Comment:
This is the wrong way to model this... again, happy to pull the PR in as long as we can clean this up after we pull the PR in.